src/callbacks_step/analysis_dg2d.jl

```diff
@@ -63,7 +63,12 @@ function calc_error_norms(func, u, t, analyzer,
   linf_error = copy(l2_error)
 
   # Iterate over all elements for error calculations
+  # Accumulate L2 error on the element first to ensure that the order of summation is the
```

Comment:
At the moment, this is only relevant for the development of the MPI capabilities for the 2D `TreeMesh`. Since I am not sure whether we need this in the long run, I wanted to keep the changes to a minimum for now. We could create an issue for this to track it, if you think it would be useful.

Reply:
Yes, that would be helpful to remind us to either revert this change or propagate it throughout Trixi.jl. Does it change anything?

Reply:
I don't think it does.

@lchristm can you please create an appropriate issue that also references this conversation?

Reply:
Created an issue (#866).

Reply:
Thanks!